test/e2e/framework/network/utils.go

```diff
@@ -91,45 +91,45 @@ const (
 // NetexecImageName is the image name for agnhost.
 var NetexecImageName = imageutils.GetE2EImage(imageutils.Agnhost)
 
-// Use functional options to configure the NetworkingTest object
+// Option is used to configure the NetworkingTest object
```

Comment:
squash the golint fixes into the correct commits?

Reply:
yep